mongomock/aggregate.py

```diff
@@ -539,6 +540,8 @@ def _handle_string_operator(self, operator, values):
 
     def _handle_date_operator(self, operator, values):
         out_value = self.parse(values)
+        if operator == '$dateTrunc':
+            return out_value['date']
```

Comment:
Apparently this isn't doing much: please add some test cases, at least the examples here https://www.mongodb.com/docs/manual/reference/operator/aggregation/dateTrunc/, and make sure to correct here.